lib/db.c

```diff
@@ -13323,8 +13321,15 @@ grn_obj_unlink(grn_ctx *ctx, grn_obj *obj)
                               obj,
                               current_lock,
                               current_reference_count);
+      grn_rc rc = GRN_SUCCESS;
       if (current_lock == 0) {
-        grn_obj_close(ctx, obj);
+        rc = grn_obj_close(ctx, obj);
```

Comment:
```suggestion
        grn_rc rc = grn_obj_close(ctx, obj);
```

数行上の宣言をなくしてここでやればよさそうです。

Reply:
あぁ。修正し忘れました。。。ありがとうございます。
なおしました。